engine/classes/ElggEntity.php

```diff
@@ -1469,7 +1469,7 @@ public function getSites($options = array()) {
 	 * @return bool
 	 */
 	public function isFullyLoaded() {
-		return ! ($this->tables_loaded < $this->tables_split);
+		return (bool)$this->guid;
```

Comment:
I don't know why this change is needed, but I'd planned to do it in #9425 anyway.